.github/workflows/lint-go.yaml

```diff
@@ -1,10 +1,7 @@
 name: Lint
 
 on:
-  push:
-    branches:
-      - master
-  pull_request:
+  workflow_call:
```

Comment:
Would be nice to have a `# Should only be used by other workflows` or something similar at the start of the file.

Reply:
Yepp, adding in next commit